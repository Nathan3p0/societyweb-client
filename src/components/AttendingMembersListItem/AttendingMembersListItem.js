import React from 'react';

const AttendingMembersListItem = (props) => {
    return (
        <li className="tableRow">
            <div className="col col-1" data-label="Member Name">{props.name}</div>
            <div className="col col-2" data-label="Member Phone">{props.phone}</div>
            <div className="col col-3" data-label="Member Email">{props.email}</div>
            <div className="col col-4" data-label="RSVP Status">{props.attending}</div>
        </li>
    );
}

export default AttendingMembersListItem;