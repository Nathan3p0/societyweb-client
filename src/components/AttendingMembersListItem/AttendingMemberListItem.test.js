import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import AttendingMembersListItem from './AttendingMembersListItem'

describe('Attending Member List Item Component Rendering Tests', () => {

    it('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<AttendingMembersListItem />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders to UI as expected', () => {
        const tree = renderer.create(<AttendingMembersListItem />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})