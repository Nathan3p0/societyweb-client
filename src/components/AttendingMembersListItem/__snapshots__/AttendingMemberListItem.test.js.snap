// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attending Member List Item Component Rendering Tests renders to UI as expected 1`] = `
<li
  className="tableRow"
>
  <div
    className="col col-1"
    data-label="Member Name"
  />
  <div
    className="col col-2"
    data-label="Member Phone"
  />
  <div
    className="col col-3"
    data-label="Member Email"
  />
  <div
    className="col col-4"
    data-label="RSVP Status"
  />
</li>
`;
